### ISSUE TITLE
[8.x] Cypress fails to start FTR configuration problem

### ISSUE THREAD

**Issue by liza-mae** [elastic/elastic-stack-testing#1177]
Currently the cypress security solution tests are blocked from running in CI on branch 8.x with the following error:

```
16:16:53 [2022-03-16 22:16:53] [INFO]  -> Running upgrade security solution tests, run 1 of 1 
16:16:53  debg KIBANA_CI_STATS_CONFIG environment variable not found, disabling CiStatsReporter
16:16:56  debg Loading config file from "/var/lib/jenkins/workspace/elastic+estf-cloud-upgrade-tests/JOB/upgradeTest2j/TASK/kibana_upgrade_tests/node/ess-testing/ci/upgrade/kibana/build/kibana/x-pack/test/security_solution_cypress/upgrade_config.ts"
16:16:56  debg Loading config file from "/var/lib/jenkins/workspace/elastic+estf-cloud-upgrade-tests/JOB/upgradeTest2j/TASK/kibana_upgrade_tests/node/ess-testing/ci/upgrade/kibana/build/kibana/test/common/config.js"
16:16:57 ERROR ValidationError: "servers.elasticsearch.auth" must be a string
16:16:57 [2022-03-16 22:16:57] [ERROR] Upgrade Security Solution Tests failed! 
16:16:57 
16:16:57 > Task :kibana:run_kibana_tests FAILED
```

This is coming from the FTR config validation and I don't see the error when I run the cypress locally, it is unclear to me the difference that would cause this error in CI environment but not local.

In any case we have attempted to solve this issue before here: https://github.com/elastic/kibana/pull/105095 but I believe this ended up being reverted by the security solutions team at one point.

**Comment by liza-mae** [comment]
@MadameSheema I think I found the problem, they recently made some changes to the ```configuration verification in FTR``` for ```8.x``` branches, that is why we needed the fix for: https://github.com/elastic/kibana/issues/127096 although it seems to be a partial revert of https://github.com/elastic/kibana/pull/105095 it is required otherwise we will get another verification error: ```ERROR ValidationError: "servers.elasticsearch.port" is required```. 

The reason it fails in CI is the environment setup from ```kibana/build.gradle``` for the ```TEST_ES_URL``` and ```TEST_KIBANA_URL```: 

https://github.com/elastic/elastic-stack-testing/blob/5e04adadc3b7bcab07a8abbc91b9d4822359098e/ci/upgrade/kibana/build.gradle#L165

https://github.com/elastic/elastic-stack-testing/blob/5e04adadc3b7bcab07a8abbc91b9d4822359098e/ci/upgrade/kibana/build.gradle#L164
 
Although that works in ```7.17.x```, it breaks in ```8.x``` most likely due to the ```FTR new enforcement of the configuration validation```. 

So I believe the fix is to add the ```username``` and ```password``` to ```TEST_ES_URL``` and ```TEST_KIBANA_URL``` and it should pass FTR configuration verification and start cypress testing.  Doing this does not appear to impact the cypress test themselves, so the same code should not break 7.17.x, but for full confirmation we need to run in CI.  I will make the updates, kick off a CI build and we will wait for the CI results to confirm and hopefully this issue will be resolved :crossed_fingers:

**Comment by liza-mae** [comment]
CI results confirmed this issue is fixed with #1194
